index.js

```diff
@@ -1,4 +1,14 @@
 "use strict";
+const currentYear = new Date().getFullYear();
+const mainHeading = document.getElementById('mainTitle');
+if (mainHeading) {
+  mainHeading.textContent = `Summer dashboard ${currentYear}`;
+}
+const pageTitle = document.getElementById('pageTitle');
+if (pageTitle) {
+  pageTitle.textContent = `Summer dashboard ${currentYear}`;
+}
+
 const fadderukaStartDate = new Date("Aug 11, 2025 00:00:00");
```

Comment:
Nice!! Kunne kanskje gjort noe à la

```suggestion
"use strict";
const currentYear = new Date().getFullYear();
const canonicalHeader = `Summer dashboard ${currentYear}`;
document.getElementById('mainTitle')?.textContent = canonicalHeader;
document.getElementById('pageTitle')?.textContent = canonicalHeader;
const fadderukaStartDate = new Date("Aug 11, 2025 00:00:00");
```

for å spare noen linjer, men details details

Reply:
🤪 
https://github.com/orjahren/summer-dashboard/commit/280c3e36c081d62aaa1914f3210386a8b1ab3446